pages/help-center/errors/serialization.mdx

```diff
@@ -1,53 +1,101 @@
 ---
-title: Serialization errors
-description: Learn about serialization errors as well as troubleshooting best practices for serialization errors in our all-in-one documentation page at Memgraph. 
+title: Conflicting transaction and serialization error
+description: Learn about conflicting transaction and serialization errors as well as troubleshooting best practices for serialization errors on our all-in-one documentation page at Memgraph. 
 ---
 
 import { Callout } from 'nextra/components'
 
-# Serialization error
+# Conflicting transaction and serialization error
 
 <Callout type="info">
 If you are encountering issues with serialization errors, we're here to help. Join the conversation on <a href="https://www.discord.gg/memgraph" target="_blank">Discord</a> for support.
 </Callout>
 
-## Error
+## Error messages
 
 1. [Unable to commit due to serialization error.](#error-1)
+2. [Cannot resolve conflicting transactions.](#error-1)
 
-## Unable to commit due to serialization error [#error-1]
+## Understand the conflicting transaction and serialization error [#error-1]
 
-Serialization errors usually occur when there are concurrent data imports and
+Conflicting transaction and serializations is in a sense the same error, that 
+usually occurs when there are concurrent data imports and
 transactions try to work on the same data points at the same time. This
```

Comment:
Conflicting transactions and serializations are, in a sense, the same error that usually occurs when concurrent data imports and transactions try to work on the same data points simultaneously. This situation can occur in both testing and production environments. It's important that your code anticipates and handles these errors.